htlcswitch/hop/payload.go

```diff
@@ -399,11 +418,18 @@ func validateBlindedRouteTypes(parsedTypes tlv.TypeMap,
 		record.EncryptedDataOnionType: {},
 		record.BlindingPointOnionType: {},
 	}
+	var requiredTLVs []tlv.Type
```

Comment:
Would it be cleaner to repurpose `allowedTLVs` to somehow cover both allowed and required TLVs?